helpdesk_mgmt_ticket_invoice/views/account_move_view.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="account_move_form_view" model="ir.ui.view">
+        <field name="name">account.move.form.view</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_move_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//button[@name='button_draft']" position="after">
+                <button name="create_ticket"
+                        type="object"
+                        string="Create ticket"
+                        groups="helpdesk_mgmt.group_helpdesk_user"
+                        attrs="{'invisible': [('type', 'not in', ['out_invoice','in_refund'])]}"
```

Comment:
That's good, but I've realized that is still incomplete. Please add another condition: if invoice is not posted (`state=posted`) this button should be invisible too